VaticanBAV/et/BAVet131.xml

```diff
@@ -27,52 +27,1023 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <textLang mainLang="gez"/>
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i11">
+                                <locus from="101v" to="106r"/>
+                                <title type="complete" xml:lang="gez" ref="LIT4860Litanies">ሊጦን፡ ዘነገሮሙ፡ ለሐዋርያት፡ እግዚእነ፡</title>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    ወካዕበ፡ ናስተበቍዕ፡ ዘኵሎ፡ ይእኅዝ<supplied reason="omitted">፡</supplied> እግዚአብሔር፡ አብ፡ ለእግዚእ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፨ እንዘ፡
+                                    <sic>ነአኵቶ፡</sic> እስመ<supplied reason="omitted">፡</supplied> ዓቀበነ፡ በኑኃ፡ ሌሊት፡ ወአብስሐነ፡ እምጽልመት፡ ውስተ፡ ብርሃን፡ እሙስና፡ ውስተ፡
+                                    ዘኢይማስን፡ እምኢያእምሮ፡ ውስተ፡ አእምሮ፡ አማን፨ 
+                                </incipit>
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i12">
+                                <locus from="106r" to="108r"/>
+                                <title type="complete" xml:lang="gez" ref="LIT2605Zayena">ዘይነግሥ፡ ዘአቡነ፡ ያሬድ፡</title>
+                                <textLang mainLang="gez"/>
+                                <explicit xml:lang="gez">
+                                    በአተነ፡ ወፀዓተነ፡ በሰማ፡ ዕቀብ፡ ከመ፡ በዳኅና፡ ወበዕረፍት፡ ሕይወተ፡ እንዘ፡ ነሃዩ፡ በኵሉ፡ አምልኮ፡ ሠናይ፡ ወበፈሪሃ፡ እግዚአብሔር፡ ወበኂሩት፡ ንትረከብ፡
+                                    በውህድ፡ ወክድከ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ተፈጸመ፡ ዘይነግሥ።
+                                </explicit>
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i13">
+                                <locus from="108r" to="110r"/>
+                                <title type="complete" xml:lang="gez" ref="LIT4270Salota">Prayer before the reading of the Miracles of Mary</title>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    <supplied reason="omitted">እግ</supplied>ዚአብሔር፡ እግዚኦ፡ ኢየሱስ፡ ክርስቶስ፡ አምላክነ፡ ዘትቤሎ፡ ለፍቁርከ፡ ዮሐንስ፡ እንዘ፡ ሀሎከ፡ ዲበ፡ ዕፀ፡ መስቀል፡ 
+                                    ቅዱስ፡ ወትቤላ፡ ለማርያም፡ ነዋ፡ ወልድኪ፡ ወለረድእከኒ፡ ነያ፡ እምከ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> 
+                                    ንስዕለከ፡ ወናስተበቍዓከ፡ በእንተ፡ ማርያም፡ ወላዲትከ፡ ወበእንተ፡ ዮሐንስ፡ ዘረፈቀ፡ ውስተ፡ ሕፅንከ፡ ወበእንተ፡ ኵሎሙ፡ ሐዋርያት፡ አኃዊከ፡ ከመ፡
+                                    <choice><corr resp="PRS4805Grebaut PRS9530Tisseran">ትረስየነ፡</corr><sic>ትረስየኒ፡</sic></choice> ድልዋነ፡ ለሰሚዓ፡ ተአርሚሃ፡ ለማርያም፡
+                                </incipit>
+                                <note>
+                                    The text is followed on  <locus target="#110r"/> by a short passage referring to the offices in <ref target="#ms_i1.6 #ms_i1.7 #ms_i1.8"/>, respectively with the
```

Comment:
These additions do not refer to "offices" but to the collections of different school chants